src/alfasim_score/converter/alfacase/_tests/test_output_results.py

```diff
@@ -1,27 +1,22 @@
-import json
-import pandas as pd
-import pytest
-from barril.units import Scalar
 from pathlib import Path
 from pytest_regressions.file_regression import FileRegressionFixture
 
 from alfasim_score.common import AnnulusLabel
-from alfasim_score.converter.alfacase.score_output_generator import ScoreOutputGenerator
+from alfasim_score.converter.alfacase.alfasim_score_converter import AlfasimScoreConverter
 
 
 def test_generate_output_file_results(
     shared_datadir: Path, datadir: Path, file_regression: FileRegressionFixture
 ) -> None:
     alfasim_results_directory = shared_datadir / "case.data"
```

Comment:
I'd rather change a bit the parameter names:
- `alfasim_results_directory`-> `alfasim_results_path`
- `score_input_filepath` -> `score_input_file`
- `output_filepath`-> `output_file`

Reply:
Ok, changed.